src/stateManager/StateManager.ts

```diff
@@ -657,8 +657,14 @@ class StateManager {
         if (!block.didEveryoneSign(participants)) {
             this.p2pEventHooks.onPostingCalldata?.();
 
+            const maxTimestamp =
+                Clock.getTimeInSeconds() +
```

Comment:
maxTimestamp = previousRelevantTimestamp+p2p+agreement+fallback

Reply:
fixed, it was due to multiple setTimouts polluting each other between tests